src/devices/chihiro/MediaBoard.h

```diff
@@ -58,8 +58,9 @@ typedef struct {
     uint32_t unknown2;
     uint16_t year;              // 0x28
     uint8_t month;              // 0x2A
-    uint8_t day;                // 0x2B
-    uint8_t unknown3[2];
+    uint8_t day;                // 0x2B
+    uint8_t videomode;          // unknown, includes resolutions and horizontal/vertical

```

Comment:
Maybe stick to the convention that seems to be applied here, and note 0x2C?

Reply:
Pushed the fix